carbonmark/pages/portfolio/retire/[project_token_0x]/index.tsx

```diff
@@ -0,0 +1,58 @@
+import { Retire, RetirePageProps } from "components/pages/Retire";
+import { loadTranslation } from "lib/i18n";
+import { getProjectInfoFromPolygonBridgedCarbon } from "lib/retireQueries";
+import { GetStaticProps } from "next";
+import { ParsedUrlQuery } from "querystring";
+
+export const getStaticPaths = async () => {
+  return {
+    paths: [],
+    fallback: "blocking",
+  };
+};
+
+interface Params extends ParsedUrlQuery {
+  project_token_0x: string;
+}
+
+export const getStaticProps: GetStaticProps<RetirePageProps, Params> = async (
```

Comment:
For this page I would flip the URL structure to be:
`carbonmark/pages/portfolio/[project_token_0x]/retire/index.tsx`

Because we might some day add 'transfer' or some other actions / views for each asset beyond retire.